### ISSUE TITLE
Inline images failing

### ISSUE THREAD

**Issue by mphelp** [sendgrid/sendgrid-python#951]
@thinkingserious and other great package maintainers:

### Inline image CID troubles!

I am able to successfully attach the PNG image. However, I cannot get it to appear inline in Gmail, Outlook, or iCloud shows the image alt. The emails are sent successfully otherwise. 

### Mail obj dump
```python
...
'content': 
    [{'type': 'text/html', 
      'value': '<h3> Forecast wape on scn unt qy by gtin no</h3><img src="cid:viz_0" alt="Inline picture of  
                  Forecast wape on scn unt qy by gtin no"/>'}], 
       'attachments': 
              [{'content': '...', 'type': 'image/png', 
               'filename': '_Forecast_wape_on_scn_unt_qy_by_gtin_no.png', 'content_id': 'viz_0'}]}
...
```

I was following THIS documentation: https://sendgrid.com/blog/embedding-images-emails-facts/

### Technical details:
* sendgrid-python version: latest 6.4.7
* python version: 3.7

**Comment by eshanholtz** [comment]
Hi @mphelp 

According to that documentation, it warns that using Content ID Inline images does not always work for web-based email clients, like Gmail. The next section, _Inline Embedding (Base64 Encoding)_, provides directions on how to attach an image inline using the Base64 encoding of that image. I recommend trying this method, since Content ID images are not working for you on those email clients.

**Comment by mphelp** [comment]
@eshanholtz 
It turns out that I wasn't including the "disposition" and it defaults to attachment and not inline. After setting it to "inline," the images show up inline in Gmail, Outlook, and iCloud!

`attachment.disposition = Disposition('inline')`

Note: The attachment use case in this repo doesn't cover inline attachment. It'd be good to include one. Do you agree? If so I could provide a code sample that matches the other use case examples.

**Comment by eshanholtz** [comment]
I agree that would be a very helpful example to include. Please submit a PR for review! Thanks!
